lib/core/src/display/shape/glyph/font.rs

```diff
@@ -22,17 +24,23 @@ use std::collections::hash_map::Entry::Vacant;
 /// Each distance and transformation values are expressed in normalized coordinates, where
 /// (0.0, 0.0) is initial pen position for an character, and `y` = 1.0 is _ascender_.
 ///
-/// `from_base_layout` transforms the _base square_ for a character, such the glyph will be rendered
-/// correctly with assigned MSDF texture. The _base square_ corners are (0.0, 0.0), (1.0, 1.0).
-/// See also `glyph_render::GLYPH_SQUARE_VERTICES_BASE_LAYOUT`.
+/// `offset` and `scale` transforms the _base square_ for a character, such the glyph will be
+/// rendered correctly with assigned MSDF texture. The _base square_ corners are (0.0, 0.0),
+/// (1.0, 1.0).
 ///
 /// For explanation of various font-rendering terms, see
 /// [freetype documentation](https://www.freetype.org/freetype2/docs/glyphs/glyphs-3.html#section-1)
 #[derive(Debug)]
 pub struct GlyphRenderInfo {
-    pub msdf_texture_rows : std::ops::Range<usize>,
-    pub from_base_layout  : nalgebra::Projective2<f64>,
-    pub advance           : f64
+    /// An index of glyph in a msdf texture (counted from the top of column. For details, see
+    /// MsdfTexture documentation.
+    pub msdf_texture_glyph_id: usize,
+    /// A required offset of the _base square_ see structure documentation for details.
+    pub offset               : Vector2<f32>,
+    /// A required scale of the _base square_ see structure documentation for details.
+    pub scale                : Vector2<f32>,
+    /// An advance value. Advance is a font-rendering specific term.
```

Comment:
This doc is hilarious :D Please, explain it just a little bit here. I want to understand it when reading the code!